### DISCUSSION TITLE
handling of 'superseded' links

### DISCUSSION THREAD

**Discussion by ckolumbus** [thomvaill/log4brains#22] [Feedback]
After finding and enjoying this great software because it was REALLY easy to adopt it for our existing ADR directories the "superseded by" handling caught me off-guard. It took me a while to find out how it is supposed to work, only the test give the final clue that the link name has to be identical to the filename of the adr markdown file (without extension). 

Could you give some explanation regarding this design decision? I read the rational for the [slug based identification](https://github.com/thomvaill/log4brains/blob/master/docs/adr/20201016-use-the-adr-slug-as-its-unique-id.md). But for the `superseded by` links I'd rather have more freedom for the link name.

**Comment by thomvaill** [comment]
Hi @ckolumbus 

Thank you for your feedback! I am really happy to hear that it was easy to make Log4brains work with your existing ADRs.
That was one of my goals.

Regarding the superseded links, you are totally right!
I made this choice to reduce the technical complexity, but I agree this is too restrictive. Especially when you have already existing ADRs like you.

I created an issue to fix this: https://github.com/thomvaill/log4brains/issues/35

Thanks a lot!

**Reply by ckolumbus** [reply]
Great to hear from you. It's a great project and I was afraid you can't find any more time to work on this!
I'm not a frontend/nodejs developer, so my active support will be limited. But we'll provide at least feedback from our usage scenarios.

Great project! Thanks for spending your time on this!
Chris

**Reply by thomvaill** [reply]
Thanks!
I would be very happy to hear your feedback!
